### PR TITLE
Add renrutnairb recommendations

### LINKED ISSUE #8: renrutnairb checklist
- [x] Create a branch
- [x] Add a file
- [ ] Make a PR
- [ ] Merge

:neckbeard:

### PR DESCRIPTION
**Pull request by brian-turner**
Adding recommendations near Prescott, WI.

Closes #8

### CODE REVIEW

File: renrutnairb-rec.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Prescott, WI
+
+### Places to Eat
+- Mama Maria's (Hudson, WI) :house_with_garden:
```
Comment (reed-helgemo) [inline renrutnairb-rec.md]: How far is this from Securian?

Comment (reed-helgemo) [inline renrutnairb-rec.md]: What's your favorite menu item at Mama Maria's?

**Review by reed-helgemo — COMMENTED** [review]
Please reply to outstanding questions

**Review by reed-helgemo — CHANGES_REQUESTED** [review]
Please see outstanding questions

File: renrutnairb-rec.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Prescott, WI
+
+### Places to Eat
+- Mama Maria's (Hudson, WI) :house_with_garden: (Recommend the Italian Sampler) - About 25-30 minutes from Securian
```
Comment (brian-turner) [inline renrutnairb-rec.md:4]: @reed-helgemo Added menu recommendation and distance from Securian.

File: renrutnairb-rec.md [hunk 3]
```diff
@@ -2,3 +2,9 @@
 
 ### Places to Eat
 - Mama Maria's (Hudson, WI) :house_with_garden: (Recommend the Italian Sampler) - About 25-30 minutes from Securian
+
+### Places to Visit
+- Ellsworth Coop Creamery (Ellsworth, WI )
+
+### Things to Do
+- Go Hunting
```
Comment (brian-turner) [inline renrutnairb-rec.md:10]: @reed-helgemo Added a place to visit and something to do.

**Review by reed-helgemo — APPROVED** [review]
Boom, Approved

**Pull request merged**